src/V3LinkDot.cpp

```diff
@@ -1986,6 +1986,24 @@ class LinkDotResolveVisitor final : public VNVisitor {
             refp->user5p(nodep);
         }
     }
+    bool paramSkip(const AstNode* nodep) {
```

Comment:
This function could use some comments as to why it's necessary. Also maybe a slightly more descriptive name?